test/integration/endpoints/fetch_messages.test.js

```diff
@@ -154,4 +243,78 @@ describe('fetch messages endpoints', () => {
       }
     );
   });
+
+  it('supports metadata', (done) => {
+    const channel = PubNub.generateUUID();
+    const expectedMessagesCount = 10;
+
+    publishMessagesToChannel(pubnub, expectedMessagesCount, channel, (messages) => {
+      pubnub.fetchMessages({ channels: [channel], count: 25, includeMeta: true }, (status, response) => {
+        const channelMessages = response.channels[channel];
+
+        assert.deepEqual(channelMessages[0].meta, { time: messages[0].message.time });
+        assert(!channelMessages[1].meta);
+        done();
+      });
+    });
+  }).timeout(60000);
+
+  it('throws when requested actions for multiple channels', () => {
+    let errorCatched = false;
+
+    try {
```

Comment:
FWIW, instead of a `try/catch` here with a flag you can use `assert.throws` => https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message

It would look something like:

```
assert.throws(
  pubnub.fetchMessages( ...),
  (error) => {
    assert.equal(error.message, '...');
  },
  'unexpected error'
);
```

Reply:
@davidnub I've tried `assert.throws` before using `try/catch` and it made something strange (actually I never used NodeJS test tools):

```
1) fetch messages endpoints throws when requested actions for multiple channels:
     TypeError: History can return actions data for a single channel only. Either pass a single channel or disable the includeMessageActions flag.
```
^
This was output and test marked as failed.